networking/eth2/src/main/java/tech/pegasys/artemis/networking/eth2/gossip/BlockGossipManager.java

```diff
@@ -32,8 +33,10 @@
   public BlockGossipManager(
       final GossipNetwork gossipNetwork,
       final EventBus eventBus,
-      final BlockValidator blockValidator) {
-    final BlockTopicHandler topicHandler = new BlockTopicHandler(eventBus, blockValidator);
+      final BlockValidator blockValidator,
+      final RecentChainData recentChainData) {
+    final BlockTopicHandler topicHandler =
```

Comment:
We'll need to wait for the store to initialize (via `recentChainData.subscribeStoreInitialized()`) so we can get a correct ForkDigest value.  Otherwise, we may end up setting up meaningless topics for the "pre-genesis" fork digest.

Reply:
Makes sense. Done.

Reply:
Done.